internal/pkg/pct/build_test.go

```diff
@@ -114,15 +129,80 @@ func TestBuild(t *testing.T) {
 				mockTemplateDir,
 				filepath.Join(mockTemplateDir, "content"),
 			},
-			mockFiles: []string{
-				filepath.Join(mockTemplateDir, "pct-config.yml"),
+			mockFiles: map[string]string{
+				filepath.Clean(filepath.Join(mockTemplateDir, "pct-config.yml")): `---
+template:
+  id: builder
+  author: puppetlabs
+  version: 1.0.0
+`,
 			},
 			tarFile:          "/path/to/nowhere/pkg/nowhere.tar",
 			gzipFile:         "/path/to/nowhere/pkg/nowhere.tar.gz",
 			expectedFilePath: "/path/to/nowhere/pkg/nowhere.tar.gz",
 			wantErr:          false,
 			mockTarErr:       false,
 		},
+		{
+			name: "Should complain that `id` is missing from pct-config.yml",
+			args: args{
+				templatePath: mockTemplateDir,
+				targetDir:    mockTemplateDir,
+			},
+			mockDirs: []string{
+				mockTemplateDir,
+				filepath.Join(mockTemplateDir, "content"),
+			},
+			mockFiles: map[string]string{
+				filepath.Clean(filepath.Join(mockTemplateDir, "pct-config.yml")): `---
+template:
+  author: puppetlabs
+  version: 1.0.0
+`,
+			},
+			wantErr:    true,
```

Comment:
I know the convention was set before your change, but might be nice to now also check the content of the error message (or raise a ticket to add this later)

Reply:
Yeah, I was lazy. Thanks. Fixing